crates/sui-framework/sources/coin.move

```diff
@@ -131,27 +131,27 @@ module sui::coin {
     /// Create a coin worth `value`. and increase the total supply
     /// in `cap` accordingly.
     public fun mint<T>(
-        value: u64, cap: &mut TreasuryCap<T>, ctx: &mut TxContext,
+        cap: &mut TreasuryCap<T>, value: u64, ctx: &mut TxContext,
```

Comment:
Let's document the convention on ordering mutable, primitive txContext etc

Reply:
We're working on it. Currently it is meant, but the tricks and tips and conventions page is incoming.